yarn.lock

```diff
@@ -2307,7 +2307,7 @@ __metadata:
     "@types/ansi-align": "npm:3.0.5"
     "@types/eslint": "npm:^8.56.1"
     "@types/node": "npm:^20.10.6"
-    "@types/react": "npm:^18.2.55"
+    "@types/react": "npm:^19.1.8"
```

Comment:
@UncleClapton I don't think these yarn.lock changes are needed from your changes, it is causing issues running CLI from repo with ink dependency.

Reply:
we found out this ties to react version compatibility with ink and other pieces. @UncleClapton Let's remove these changes to package.json/yarn.lock and we can revisit these in separate PR